dts/arm/nordic/nrf52810.dtsi

```diff
@@ -0,0 +1,72 @@
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/i2c/i2c.h>
+#include <nordic/mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4";
+			reg = <0>;
+		};
+	};
+
+	flash-controller@4001E000 {
+			compatible = "nrf,nrf52-flash-controller";
+			reg = <0x4001E000 0x550>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			label="NRF_FLASH_DRV_NAME";
+
+			flash0: flash@0 {
+				compatible = "soc-nv-flash";
+				label = "NRF_FLASH";
+				reg = <0x00000000 DT_FLASH_SIZE>;
+				write-block-size = <4>;
+			};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 DT_SRAM_SIZE>;
+	};
+
+	soc {
+		uart0: uart@40002000 {
+			compatible = "nordic,nrf-uarte";
+			reg = <0x40002000 0x1000>;
+			interrupts = <2 1>;
+			status = "disabled";
+			label = "UART_0";
+		};
+
+		i2c0: i2c@40003000 {
+			compatible = "nordic,nrf5-i2c";
```

Comment:
nRF52810 has only TWIM and no TWI that is present on nRF51 and other nRF52s. Shouldn't a new binding ("nordic,nrf-twim") be introduced and this value changed?